dts/sifive-hifive1-revb.dts

```diff
@@ -0,0 +1,254 @@
+/dts-v1/;
```

Comment:
There is already one in freedom-e-sdk. Having another will only create mismatch and confusion. Remove this

Reply:
removed.